src/SplitPackages/SplitPackages.csproj

```diff
@@ -4,9 +4,10 @@
   <PropertyGroup>
     <Description>Copies NuGet packages form a given source folder into a specific set of folders based on a CSV file.</Description>
     <VersionPrefix>1.0.1</VersionPrefix>
-    <TargetFrameworks>netcoreapp1.0;net451</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.0</TargetFrameworks>
     <DebugType>portable</DebugType>
     <OutputType>exe</OutputType>
+    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
```

Comment:
We aren't doing this with the verifier (https://github.com/aspnet/BuildTools/blob/dev/src/NuGetPackageVerifier/NuGetPackageVerifier.csproj#L8). What's different here?